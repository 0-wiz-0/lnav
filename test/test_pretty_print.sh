#! /bin/bash

echo '2015-04-18T13:16:30.003 198.51.100.45 <foo>98.51.100.154</foo>9 198.158.100.4<1054 198.51.100.1546 544.9.8.7 98.542.241.99 19143.2.5.6' | \
    run_test ${lnav_test} -n -c ":switch-to-view pretty"

check_output "pretty print not able to properly grok ipv4?" <<EOF
2015-04-18T13:16:30.003 198.51.100.45 (unknown)
<foo>98.51.100.154 (unknown)</foo>
9 198.158.100.4 (unknown)<1054 198.51.100.1546 544.9.8.7 98.542.241.99 19143.2.5.6
EOF
