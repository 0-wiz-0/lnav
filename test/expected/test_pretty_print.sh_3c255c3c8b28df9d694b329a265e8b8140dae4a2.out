{
    "wrapper": [
            {"message":""
            select Id from Account where id = $sfid
                                             ^
            ERROR at Row:1:Column:34
            line 1:34 no viable alternative at character '$'
""}]}
